src/selftest/selftest_clockEvents.c

```diff
@@ -119,6 +110,25 @@ void Test_ClockEvents() {
 		SELFTEST_ASSERT_CHANNEL(2, 20);
 		SELFTEST_ASSERT_CHANNEL(3, 30);
 	}
+
+	// Test with calculated times
+	ResetEventsAndChannels(3);
+
+	CMD_ExecuteCommand("setChannel 4 13", 0);
+	CMD_ExecuteCommand("setChannel 5 55", 0);
+	CMD_ExecuteCommand("setChannel 6 $CH4*3600+54*60+59", 0);
+
+	CMD_ExecuteCommand("addClockEvent $CH6 0xff 4 backlog addChannel 1 10; echo test event for 4", 0);
+	//CMD_ExecuteCommand("addClockEvent $CH4:$CH5 0xff 5 backlog addChannel 2 20; echo test event for 5", 0);
```

Comment:
To allow this, it is not so simple as enabling the `TOKENIZER_ALTERNATE_EXPAND_AT_START` flag for the string tokenizer. If we wish to pursue this notation, there would need some modification in the tokenizer code.